lib/looker-sdk/client.rb

```diff
@@ -238,6 +238,32 @@ class Serializer < Sawyer::Serializer
       def encode(data)
         data.kind_of?(Faraday::UploadIO) ? data : super
       end
+
+      # slight modification to the base class' decode_has_value function to
+      # less permissive when decoding time values.
+      #
+      # See https://github.com/looker/helltool/issues/22037 for more details
```

Comment:
Link to private helltool issue is not appropriate here. Please remove it. Ideally, file a short issue in *this* public repo that has a succinct problem statement.